docs/content/integrations/dbt/quickstart.mdx

```diff
@@ -41,6 +41,28 @@ The `dagster-dbt` library installs both `dbt-core` and `dagster` as dependencies
 
 ---
 
+## Wrap your dbt project with Dagster
```

Comment:
I know we use the ["wrap" lingo here](https://docs.dagster.io/integrations/dbt/reference#scaffolding-a-dagster-project-from-a-dbt-project), but maybe we should consolidate on "load"

```suggestion
## Load your dbt project in Dagster
```

Reply:
Done in [6e2bb9f](https://github.com/dagster-io/dagster/pull/21816/commits/6e2bb9f11e758e37394c532832d1c995590403bb)